src/lib.rs

```diff
@@ -0,0 +1,207 @@
+pub use toml_config_derive::TomlConfig;
+pub use toml;
+
+/// Run this on your struct implementing TomlConfig to generate a
+/// serialization/deserialization test for it.
+#[macro_export]
```

Comment:
Is this needed? For me it does not seem that the tests will run with `cargo test` - I would prefer explicit tests anyways.

Reply:
I think it is nice to provide some means of testing serialization and deserialization for end users without them having to add the toml dependency to their own projects, which they would if they would write the code themselves. However, I agree that a macro isn’t strictly necessary. We could provide a function generic over T: TomlConfig + Serialize + PartialEq instead.